### DISCUSSION TITLE
alia设置不生效

### DISCUSSION THREAD

**Discussion by xgdyzc** [umijs/umi#6881] [Q&A]
utils文件夹路径为/src/utils,在.umirc.ts中配置了alias
`import { defineConfig } from 'umi';
const path = require('path');
export default defineConfig({
  nodeModulesTransform: {
    type: 'none',
  },
  routes: [
    { path: '/', component: '@/pages/ucAdmin/index' },
  ],
  alias: {
    utils: '@/utils',
    components:'@/components',
  },
`

/src/pages/ucadmin/service.js如下:
`import {api,request} from /'utils/'
export async function osUserQueryUser(params,config) {
  return request.get(api.osUserQueryUser, params,config);
}`

build时报错如下：
`This dependency was not found:* utils in ./src/pages/ucAdmin/service.js`

**Comment by dungtran5** [comment]
Poor guy
